### ISSUE TITLE
Make the inventory window wider instead of taller while in Edit Mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [arkayenro/arkinventory#627]
```
This is an alternate solution to the problem described at 
http://code.google.com/p/arkinventory/issues/detail?id=409

When displaying Edit Mode, keep the height and width of non-empty bars (and the 
empty slots bar) the same as they appear in non-Edit Mode, then insert the 
empty bars where they belong and adjust the window width to accommodate.

This way, the Edit mode and non-Edit mode have a more similar appearance and 
the window growth will be much less than what occurs in the current 
implementation of Edit Mode.

In my case, the Bags width is 10 and I have 10 bars per row but most of them 
are empty because different characters (which all share the same settings) will 
use different bars (for example, each character will have different spec gear 
or profession category items). If any of those bars has items, then the current 
Edit Mode will just grow the window vertically by the number of items divided 
by 2. The proposed method will only grow the window width by the number of 
empty bars on a row minus the number of non-empty bars, regardless of the 
number of items.
```

Original issue reported on code.google.com by `jo...@shaw.ca` on 22 Sep 2013 at 11:17

**Comment by joevt** [comment]
This would make editing much nicer.

**Comment by joevt** [comment]
Looks like this was implemented in 3.06.12. Thanks for that. It saves a lot of vertical scrolling.

**Comment by arkayenro** [comment]
yep, added in 3.06.12
